rpm/debugedit.spec

```diff
@@ -0,0 +1,71 @@
+%define rpmhome /usr/lib/rpm
+
+Name: debugedit
+Version: 5.1
+Release: 1
+Summary: Tools for debuginfo creation
+License: GPLv3+ AND GPLv2+ AND LGPLv2+
```

Comment:
Is GPLv3+ still problem for us? And I think we do not have such License lines in use?

Reply:
With a lowercase `and` there are plenty of matches: shadow-utils, systemd, elfutils, glibc, meego-rpm-config, gcc, util-linux...

Reply:
It seems most use lowercase but some use uppercase.